packages/netlify-cms-core/src/components/Editor/EditorControlPane/EditorControlPane.js

```diff
@@ -55,6 +56,7 @@ export default class ControlPane extends React.Component {
 
     return (
       <ControlPaneContainer>
+        {collection.has('path') && <PathPreview collection={collection} entry={entry} />}
```

Comment:
Would be nice if the path preview can come immediately before the `identifier` field.

Reply:
Good idea I'll update it